src/schema/submission_centric_schema.yaml

```diff
@@ -881,6 +909,7 @@ classes:
       - has dataset
       - has data access policy
       - has data access committee
+      - has member
```

Comment:
I don't understand why this is added to the submission object.

Reply:
Well, here we can discuss if we want to have embedded Member object (so to treat it like Phenotypic Feature or Ancestry) or to include it on the submission level and reference it. Both approaches are possible. @KerstenBreuer What do you think about it?

Reply:
Is fine right now.